test/csuite/wt2909_checkpoint_integrity/main.c

```diff
@@ -52,13 +52,9 @@
  * given N may give different results on different runs. But approximate optimal N can be
  * determined, allowing a series of additional tests clustered around this N.
  *
- * The data is stored in two tables, one having indices. Both tables have the same keys and are
```

Comment:
Please rewrite to explain the two tables without mentioning indexes.

* The data is stored in two tables. Both tables have the same keys and are
* updated with the same key in a single transaction.
*